java/core/src/main/java/org/bondlib/BlobBondType.java

```diff
@@ -107,7 +107,9 @@ protected final Blob deserializeValue(TaggedDeserializationContext context) thro
     }
 
     @Override
-    protected final Blob deserializeValue(UntaggedDeserializationContext context) throws IOException {
+    protected final Blob deserializeValue(
+        UntaggedDeserializationContext context,
+        TypeDef typeDef) throws IOException {
```

Comment:
>TypeDef typeDef [](start = 8, length = 15)

For Blob and all primitive types, need to check whether the given TypeDef is compatible with the current type and appropriate conversion or raise an error. Do you plan to add this?

Reply:
Not in this commit.